core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackSyncService.java

```diff
@@ -217,6 +222,7 @@ private void handleSyncResult(Stack stack, Map<InstanceSyncState, Integer> insta
             eventService.fireCloudbreakEvent(stack.getId(), AVAILABLE.name(), STACK_SYNC_INSTANCE_STATE_SYNCED);
         } else if (isAllStopped(instanceStateCounts, instances.size()) && stack.getStatus() != STOPPED) {
             updateStackStatusIfEnabled(stack.getId(), DetailedStackStatus.STOPPED, SYNC_STATUS_REASON, stackStatusUpdateEnabled);
+            updateClusterStatusIfEnabled(stack.getId(), STOPPED, stackStatusUpdateEnabled);
```

Comment:
you should update the cluster status in the other case as well

Reply:
I don't think so

Reply:
We talked in slack. The cluster status will be set at the other part of the sync. I accept the change